features/step_definitions/govuk_component_steps.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
```

Comment:
Would it be ideal to extend this to include govuk textbox and other form inputs?

Reply:
I think so, I added [this commit](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/4080/commits/701de5ef718c1f931293e17df210e950a2bbb79e) to move other steps I found to this file. Are there others you are thinking-of/have-found?

Reply:
I was thinking more of https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/features/step_definitions/common_steps.rb#L204

Adding specificity like the other govuk steps by finding the relevant class name, but then again, I guess the reference above seem loose and robust.
What do you think?